docs/guides/setup_concord_with_env.md

```diff
@@ -0,0 +1,45 @@
+# Setting up with env variables
+
+## What is an "env variable"?
+
+Env variables are values that are dynamic-named that can affect how processes behave in case their values are edited. It can store values like paths, names, ints, and anything (except for files). Widely used in many applications.
+
+## JSON fields vs env variables
+
+Even JSON is widely used by applications for storing data, it is NOT good for storing sensitive data. And **env variables are much easier to get** than JSON fields, since you can use functions like `getenv`, and in **JSON you would need to use libraries** like [jsmn-find](https://github.com/lcsmuller/jsmn-find) to read their fields.
```

Comment:
```suggestion
Even JSON is widely used by applications for storing data, it is NOT good for storing sensitive data. And **env variables are much easier to get** than JSON fields since you can use very straightforward functions like `getenv`, while in **JSON you would require third-party libraries** such [jsmn-find](https://github.com/lcsmuller/jsmn-find) to read stored data.
```